builders/toolkit/integrations/wallets/rainbowkit.md

```diff
@@ -0,0 +1,314 @@
+    
+ - To follow this guide, visit [Next.js](https://nextjs.org/docs){target=\_blank} and follow the **Automatic Installation** instructions, selecting Typescript and the App Router options during setup 
+
+- The wallet being used must support custom networks, as the Dancebox appchain will be added manually.
+
+- A WalletConnect `projectId` - every dApp relying on WalletConnect is required to have an associated `projectId`. It is free to create an account,  and an ID can be generated instantly.
+
+    To obtain a WalletConnect `projectId`:
+
+    1. Visit [WalletConnect Cloud](https://cloud.walletconnect.com/){target=\_blank}
+    2. On the **Projects** page, select **Create** 
+    3. Add the project information (leaving the **Homepage URL** blank if the dApp is not deployed)
+    4. Select the **AppKit** SDK
+    5. Select a coding environment or platform (React is used in this guide)
+    6. Locate the `projectId` in the left menu. or find it in the **Get started** code snippet of WalletConnect Quickstart.
+ 
+### Getting Started {: #getting-started }
+
+Ensure that the project is in the root directory before proceeding, then install RainbowKit and its peer dependencies:
+
+=== "npm"
+
+    ```bash
+    npm install @rainbow-me/rainbowkit wagmi viem@2.x @tanstack/react-query
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm install @rainbow-me/rainbowkit wagmi viem@2.x @tanstack/react-query
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn add @rainbow-me/rainbowkit wagmi viem@2.x @tanstack/react-query
+    ```
+These packages provide the core functionality for wallet connections (RainbowKit), Ethereum interactions (wagmi and viem), and state management (TanStack Query).
+
+Next, start the development server to create a local dApp instance:
+
+=== "npm"
+
+    ```bash
+    npm run dev
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm run dev
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn dev
+    ```
+
+After starting the server, open [http://localhost:3000](http://localhost:3000){target=\_blank},  to view the Next.js application in the browser.
+
+To test the RainbowKit connection, the MetaMask app can be used. Ensure that the Dancebox appchain is connected in the MetaMask wallet.
+
+To add the Dancebox appchain to MetaMask:
+
+1. Open MetaMask and go to Settings > Networks > Add Network.
+2. Input the following information:
+   - Network Name: Dancebox
+   - RPC URL: `https://fraa-dancebox-3001-rpc.a.dancebox.tanssi.network`
+   - Chain ID: 5678
+   - Currency Symbol: TANGO
+   - Block Explorer URL: `https://fra-dancebox-3001-bs.a.dancebox.tanssi.network/`
+
+For detailed instructions on connecting MetaMask to the Dancebox appchain, refer to the [Tanssi Documentation](https://docs.tanssi.network/builders/toolkit/ethereum-api/wallets/metamask/#connect-metamask-to-evm-appchain){target=\_blank}.
```

Comment:
This is the Tanssi documentation, I'd change it to "refer to a -meaningful name of the article or section within the article-"